examples/particle/main.rs

```diff
@@ -4,51 +4,110 @@ use blade_graphics as gpu;
 
 mod particle;
 
+// const SAMPLE_COUNT: u32 = 1;
```

Comment:
would be useful to support `SAMPLE_COUNT = 1` in the code here, which would mean - not creating the extra MSAA texture at all

Reply:
Usually do it that way. Should I make the SAMPLE_COUNT parameter dynamically changable in the UI as well? Should be possible but the pipeline needs recreating as well in that case

Reply:
it's a nice feature but it wouldn't block this PR from landing